### ISSUE TITLE
Overseerr: Installs development branch of the app

### ISSUE THREAD

**Issue by ThisIsJeron** [community-scripts/ProxmoxVE#321]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

After using default settings for overseerr, I am seeing within settings that it installs the development branch. I think it is best that it installs a release branch and not a development branch

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

**Comment by newzealandpaul** [comment]
Yes, you are correct it installs the main branch, and when updated, it simply pulls from the main branch. I think making it use a release would be a good idea. I will have a go at changing it.

**Comment by j-cadena1** [comment]
I am also seeing that behaviour. The last update to the develop branch on my install shows that it was over 500 days ago. Is the project still alive I wonder...

**Comment by oOStroudyOo** [comment]
Yes, it looks like Develop is their default branch. Last commit was 2 months ago.
Master, last updated July 28th 2023.

This seems more like a poorly run Overseerr repo, rather than a helper-script issue.
The helper-script currently clones the repo, which I believe by default will pull the repos default branch.

I guess this is up to the community here, would you rather pull the master which is 18 months out of date.
Or pull their "default" branch which is Develop?

**Comment by MickLesk** [comment]
Yep. It is actually the intention of the scripts to use the develop branch, as master/main has not been maintained there for ages, or only rarely.
